internal/services/users-service.go

```diff
@@ -180,6 +180,22 @@ func (s *UsersService) GetUserById(id string) (*entities.User, *customErrors.Int
 	return entities.NewUser(response.Id, response.Name, response.Email, response.Phone, response.BirthDate, string(response.Role)), nil
 }
 
+func (s *UsersService) GetUsers(where myTypes.Where) ([]entities.User, *customErrors.InternalError) {
+	if len(where) == 0 {
+		where = myTypes.Where{
+			"role": map[string]string{"eq": "admin"},
+		}
+	}
+
+	repositoryData, err := s.repository.FindAll(where)
```

Comment:
Aqui poderia só passar o `where`, não precisaria verificar o tamanho e passar por padrão `role admin`. No seu método no controller fica mais certo fazer isso.
```suggestion
	repositoryData, err := s.repository.FindAll(where)
```

Reply:
Beleza. 
Só para fins de documentação: tinha feito dessa forma pois na especificação estava "Na lógica do service por padrão só serão listados adms".